### DISCUSSION TITLE
getting a *.po files

### DISCUSSION THREAD

**Discussion by Gel0bmstu** [rhinstaller/anaconda#3355] [Q&A] [answered]
How can i get a filled ru.po files for anaconda? need to fix some specific translation for my usecase

**Comment by VladimirSlavik** [comment, marked as answer]
Easy!
1. `git clone https://github.com/rhinstaller/anaconda-l10n.git`
2. `cd anaconda-l10n/<branch>` - pick the right branch
3. `cp ru.po <somewhere>`

For our other projects, the pattern is the same - repo `<name>-l10n` has translations for project `name`. The contents of these repositories comes from [Weblate](https://translate.fedoraproject.org/) and is synced automatically.

**Reply by Gel0bmstu** [reply]
great, thanks!

**Reply by VladimirSlavik** [reply]
Actually, I made a mistake. `make -f Makefile.am po-pull` will create directory `po` that should correspond to the active branch. I will edit the answer.

And you're welcome :)
